Makefile

```diff
@@ -669,8 +669,8 @@ test-e2e-run: generate-e2e-templates install-tools ## Run e2e tests.
     	-e2e.artifacts-folder="$(ARTIFACTS)" \
     	-e2e.config="$(E2E_CONF_FILE_ENVSUBST)" \
     	-e2e.skip-log-collection="$(SKIP_LOG_COLLECTION)" \
-    	-e2e.skip-resource-cleanup=$(SKIP_CLEANUP) -e2e.use-existing-cluster=$(SKIP_CREATE_MGMT_CLUSTER) $(E2E_ARGS) ; \
-	$(MAKE) clean-release-git
+    	-e2e.skip-resource-cleanup=$(SKIP_CLEANUP) -e2e.use-existing-cluster=$(SKIP_CREATE_MGMT_CLUSTER) $(E2E_ARGS) && \
+			$(MAKE) clean-release-git
```

Comment:
just curious, is there difference between doing && vs just putting the command on a new line (without the `; \ `)?

Reply:
It looks like not according to my toy experiment, so I think that would make this a bit simpler.
```make
test-e2e-run:
	false
	$(MAKE) clean-release-git
```

```
% make test-e2e-run
false
make: *** [Makefile:667: test-e2e-run] Error 1
```

---

```make
test-e2e-run:
	true
	$(MAKE) clean-release-git
```

```
% make test-e2e-run
true
make clean-release-git
make[1]: Entering directory '/home/jon/github/capz'
git restore ./*manager_image_patch.yaml ./*manager_pull_policy.yaml
make[1]: Leaving directory '/home/jon/github/capz'
```